src/utilcode/clrhost_nodependencies.cpp

```diff
@@ -745,7 +745,7 @@ void ClrFlsAssociateCallback(DWORD slot, PTLS_CALLBACK_FUNCTION callback)
     GetExecutionEngine()->TLS_AssociateCallback(slot, callback);
 }
 
-void ** __stdcall ClrFlsGetBlockGeneric()
+void ** ClrFlsGetBlockGeneric()
 {
```

Comment:
How about using ``LPVOID *ClrFlsGetBlockGeneric()``?

``CLRFLSGETBLOCK`` is defined as follows:
```
src$ grep -Rn 'CLRFLSGETBLOCK'
inc/clrhost.h:82:typedef LPVOID* (*CLRFLSGETBLOCK)();
```